core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -121,7 +121,7 @@ public String description() {
   @Override
   public void execute(final String[] args) throws Exception {
     Opts opts = new Opts();
-    opts.parseArgs(PrintInfo.class.getName(), args);
+    opts.parseArgs("accumulo rfile-info", args);
```

Comment:
I am just changing the usage. Previously, it would print:

```bash
accumulo rfile-info -h
Usage: org.apache.accumulo.core.file.rfile.PrintInfo [options]  <file> { <file> ... }
  Options:
    -c, --config
       Comma-separated Hadoop configuration files
       Default: []
```

It now prints:

```bash
accumulo rfile-info -h
Usage: accumulo rfile-info [options]  <file> { <file> ... }
  Options:
    -c, --config
       Comma-separated Hadoop configuration files
       Default: []
```